it/test/uk/gov/hmrc/tradergoodsprofiles/controllers/GetRecordsControllerIntegrationSpec.scala

```diff
@@ -552,31 +560,6 @@ class GetRecordsControllerIntegrationSpec
       )
     }
 
-    "return bad request when lastUpdatedDate query parameter is invalid" in {
```

Comment:
I think we should keep this test case  within the integration test. It verifies the the whole flow and the expected error is the lastUpdatedDate query param is invalid.

Reply:
it mostly duplicated code to mimic the router error. I don't see a need for it.

Reply:
Alright then, Im resolving these comments